Makefile

```diff
@@ -1,6 +1,6 @@
 include golang.mk
 .DEFAULT_GOAL := test # override default goal set in library makefile
-.PHONY: test build
+.PHONY: test build hardcoded.go
```

Comment:
wasn't totally sure the best way to setup the makefile, so let me know if you have a better idea